src/pspm_options.m

```diff
@@ -195,14 +195,15 @@
     options = autofill(options, 'target',                 'screen',   '*Char'           );
   case 'extract_segments'
     % 2.22 pspm_extract_segments --
-    options = autofill(options, 'length',                 10,         '>=', 0           );
+    options = autofill(options, 'length',                 10,         '>', 0            );
     options = autofill(options, 'norm',                   0,          1                 );
     options = autofill(options, 'outputfile',             '',         '*Char'           );
     options = autofill(options, 'overwrite',              0,          1                 );
     options = autofill(options, 'plot',                   0,          1                 );
-    options = autofill(options, 'timeunit',               'seconds',  {'seconds', ...
+    options = autofill(options, 'timeunits',              'seconds',  {'seconds', ...
                                                                       'samples',...
                                                                       'markers'}        );
+    options = autofill(options, 'marker_chan_num',        'marker',   '*Num*Char'       );  
```

Comment:
This has not been available in GUI, can I have your comment about whether it is necessary to have?

Reply:
Yes, it is necessary, and the GUI will be updated in relation to #671; but before this PR and PR #666 need to be merged.